normalize_repos/normalize_repos.py

```diff
@@ -27,9 +30,19 @@ def get_cc_repos(github):
     return cc.get_repos()
 
 
+def is_engineering_project(repo):
+    contents = repo.get_contents(".cc-metadata.yml")
+    contents = contents.decoded_content
+    metadata = yaml.safe_load(contents)
+    return metadata.get("engineering_project", False)
+
+
 def update_branch_protection(repo):
     master = repo.get_branch("master")
-    if repo.name not in branch_protections.EXEMPT_REPOSITORIES:
+    if (
+        repo.name not in branch_protections.EXEMPT_REPOSITORIES
+        or not is_engineering_project(repo)
```

Comment:
Sorry for my incorrect comment earlier, but I think since we need to exempt non-engineering repos, this will be the correct condition.
```suggestion
        and is_engineering_project(repo)
```

Reply:
I also missed that logic flaw. No worries, changes done.